apps/docs/content/guides/realtime/authorization.mdx

```diff
@@ -0,0 +1,293 @@
+      and realtime.messages.extension in ('broadcast')
+  )
+);
+```
+
+### Presence
+
+The `extension` field on the `realtime.messages` table records the message type. For Presence messages, the value of `realtime.messages.extension` is `presence`. You can check for this in your RLS policies.
+
+#### Allow users to listen to Presence messages on a Channel
+
+Create a policy for `select` on `realtime.messages` where `realtime.messages.extension` is `presence`.
+
+```sql
+create policy "authenticated users can listen to presence changes in a room"
+on "realtime"."messages"
+for select
+to authenticated
+using (
+  exists (
+    select
+      ru.user_id
+    from
+      rooms_users ru
+    where
+      ru.user_id = (select auth.uid())
+      and ru.topic = (select realtime.topic())
+      and realtime.messages.extension in ('presence')
+  )
+);
+```
+
+#### Allow users to send Presence messages on a channel
+
+To update the Presence status for a user create a policy for `insert` on `realtime.messages` where the value of `realtime.messages.extension` is `presence`.
+
+```sql
+create policy "authenticated users can register and update their presence in a room"
+on "realtime"."messages"
+for insert
+to authenticated
+with check (
+  exists (
+    select
+      ru.user_id
+    from
+      rooms_users ru
+    where
+      ru.user_id = (select auth.uid())
+      and ru.name = (select realtime.topic())
+      and realtime.messages.extension in ('presence')
+  )
+);
+```
+
+### Presence and Broadcast
+
+Authorize both Presence and Broadcast by including both extensions in the `where` filter.
+
+#### Broadcast and Presence read
+
+Authorize Presence and Broadcast read in one RLS policy.
+
+```sql
+create policy "authenticated user can listen to Broadcast and Presence messages in room"
+on "realtime"."messages"
+for select
+to authenticated
+using (
+  exists (
+    select
+      ru.user_id
+    from
+      rooms_users ru
+    where
+      ru.user_id = (select auth.uid())
+      and ru.topic = (select realtime.topic())
+      and realtime.messages.extension in ('broadcast', 'presence')
+  )
+);
+```
+
+#### Broadcast and Presence write
+
+Authorize Presence and Broadcast write in one RLS policy.
+
+```sql
+create policy "authenticated user can send Broadcast and Presence messages in a room"
+on "realtime"."messages"
+for insert
+to authenticated
+with check (
+  exists (
+    select
+      ru.user_id
+    from
+      rooms_users ru
+    where
+      ru.user_id = (select auth.uid())
+      and ru.name = (select realtime.topic())
+      and realtime.messages.extension in ('broadcast', 'presence')
+  )
+);
+```
+
+## Interaction with Postgres Changes
+
+Realtime Postgres Changes are separate from Channel authorization. The `private` Channel option does not apply to Postgres Changes.
+
+When using Postgres Changes with RLS, database records are sent only to clients who are allowed to read them based on your RLS policies.
```

Comment:
We should specify which RLS.

> database records are sent only to clients who are allowed to read them based on your RLS policies.

e.g. based on RLS policies on your tables and not `realtime.messages`.